frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -573,8 +573,9 @@ export { useFlag } from '../utils/flags';
  * @param {YAMLEditorProps['toolbarLinks']} toolbarLinks - Array of ReactNode rendered on the toolbar links section on top of the editor.
  * @param {YAMLEditorProps['onChange']} onChange - Callback for on code change event.
  * @param {YAMLEditorProps['onSave']} onSave - Callback called when the command CTRL / CMD + S is triggered.
+ * @param  ref - React reference to monaco editor. For more details, please, visit https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IStandaloneCodeEditor.html.
  */
-export const YAMLEditor: React.FC<YAMLEditorProps> = require('@console/internal/components/AsyncYAMLEditor')
+export const YAMLEditor: React.ForwardRefExoticComponent<YAMLEditorProps & React.RefAttributes<any>> = require('@console/internal/components/AsyncYAMLEditor')
```

Comment:
```suggestion
export const YAMLEditor: React.ForwardRefExoticComponent<YAMLEditorProps & React.RefAttributes<MonacoEditor>> = require('@console/internal/components/AsyncYAMLEditor')
```

Reply:
To do that i should install `react-monaco-editor`. Can I do it?

Reply:
It's best to have a proper type. We could install the lib just for the type, copy the type, or make a very small focused type that provides APIs for only specific functions.

Installing the editor is fine considering this is a component that ends up using the editor anyways. Furthermore, in this case we only need to import the typescript type.

Reply:
Fixed. I added `react-monaco-editor` as a dependence